subsys/bluetooth/audio/shell/bap.c

```diff
@@ -1637,29 +1637,41 @@ static struct bt_bap_broadcast_sink_cb sink_cbs = {
 #endif /* CONFIG_BT_BAP_BROADCAST_SINK */
 
 #if defined(CONFIG_BT_AUDIO_RX)
+static size_t lost_pkts;
+static size_t err_pkts;
+static size_t dup_psn;
+static size_t rx_cnt;
+static size_t dup_ts;
+
 static void audio_recv(struct bt_bap_stream *stream,
 		       const struct bt_iso_recv_info *info,
 		       struct net_buf *buf)
 {
 	static struct bt_iso_recv_info last_info;
-	static size_t rx_cnt;
-
-	/* TODO: Make it possible to only print every X packets, and make X settable by the shell */
-	if ((rx_cnt % 100) == 0) {
-		shell_print(ctx_shell,
-			    "[%zu]: Incoming audio on stream %p len %u ts %u seq_num %u flags %u",
-			    rx_cnt, stream, buf->len, info->ts, info->seq_num,
-			    info->flags);
-	}
 
 	if (info->ts == last_info.ts) {
-		shell_error(ctx_shell, "[%zu]: Duplicate TS: %u",
-			    rx_cnt, info->ts);
+		dup_ts++;
 	}
 
 	if (info->seq_num == last_info.seq_num) {
-		shell_error(ctx_shell, "[%zu]: Duplicate seq_num: %u",
-			    rx_cnt, info->seq_num);
+		dup_psn++;
+	}
+
+	if (info->flags & BT_ISO_FLAGS_ERROR) {
+		err_pkts++;
+	}
+
+	if (info->flags & BT_ISO_FLAGS_LOST) {
+		lost_pkts++;
+	}
+
+	/* TODO: Make it possible to only print every X packets, and make X settable by the shell */
```

Comment:
Sounds like a good idea to make it configurable.  Also it seems like a fairly small change.  Could this be added in the PR? otherwise LGTM

Reply:
I think it makes sense to add to this PR :)